zerver/openapi/zulip.yaml

```diff
@@ -6648,6 +6651,23 @@ paths:
             type: boolean
             default: false
           example: true
+        - name: message_ids
+          in: query
+          description: |
+            A list of message IDs to fetch. Server will return messages matching
+            the narrow and the message IDs provided.
+
+            It is mutually exclusive with some properties like `anchor`, `num_before`
+            and `num_after`, i.e., if `message_ids` is provided, then `anchor`,
```

Comment:
This is the reference documentation, so it should be definitive about exactly which properties it's referring to here.

Reply:
Updated to remove `some properties like`.